app/models/sponsor.rb

```diff
@@ -3,9 +3,10 @@ class Sponsor < ActiveRecord::Base
   has_one :address
   has_many :sponsor_sessions
   has_many :sessions, through: :sponsor_sessions
+  belongs_to :member
```

Comment:
This should also be a many to many association, as a sponsor could have more than one points of contact.